common/lib/xmodule/xmodule/vertical_block.py

```diff
@@ -20,6 +20,8 @@
 CLASS_PRIORITY = ['video', 'problem']
 
 
+@XBlock.needs("user")
+@XBlock.needs("bookmarks")
```

Comment:
I think you might be able to [combine these two needs](https://github.com/edx/XBlock/blob/2d16fcf050c5dd0e13208672e95d8c453309efc7/xblock/test/test_runtime.py#L527) like:
```python
@XBlock.needs('user', 'bookmarks')
```